.eslintrc

```diff
@@ -44,6 +44,7 @@
     "react/prefer-stateless-function": 0,
     "react/no-array-index-key": 0,
     "react/no-unused-prop-types": 0,
+    "react/require-default-props": 0,
```

Comment:
I remember there was a reason for that, but now I am thinking it is also useful.
It will be ideal, if we can avoid ignoring this rule.